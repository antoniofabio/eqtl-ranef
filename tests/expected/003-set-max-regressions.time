	Command being timed: "./test-003-set-max-regressions.sh"
	User time (seconds): 48.87
	System time (seconds): 0.06
	Percent of CPU this job got: 99%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:48.98
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 151728
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 84488
	Voluntary context switches: 58
	Involuntary context switches: 337
	Swaps: 0
	File system inputs: 7522
	File system outputs: 7636
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
