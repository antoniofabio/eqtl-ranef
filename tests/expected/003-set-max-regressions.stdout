outcome103	regressor2	0.0350498180719516	-0.770112014194364	-2.13775711159648	100	cis
outcome11	regressor1	0.0360811130768763	0.712108746845099	2.12554519231098	100	cis
outcome130	regressor2	0.0532806723361843	0.74953154272858	1.95651087829599	100	cis
outcome142	regressor2	0.0926532230520083	0.664084271814361	1.69831926734315	100	cis
outcome144	regressor2	0.0894860221627168	0.563228461727169	1.71528390583076	100	cis
outcome147	regressor2	0.0638026882190459	-0.704636023338127	-1.87498838948518	100	cis
outcome14	regressor2	0.0729166665363878	0.634179086564488	1.81305151210422	100	cis
outcome152	regressor1	0.00486033966017864	0.937232714793556	2.88237536590414	100	cis
outcome159	regressor2	0.0859586247227335	0.623252432949747	1.73476601340515	100	cis
outcome164	regressor2	0.0546433813897824	0.780391783333929	1.94522411506966	100	cis
outcome165	regressor1	0.0626949002100936	-0.673063797231648	-1.88301334967149	100	cis
outcome166	regressor2	0.0562098675926978	0.730220904676727	1.9325403698335	100	cis
outcome169	regressor1	0.0656526547593042	-0.593534848437114	-1.86184379165077	100	cis
outcome199	regressor1	0.037511496356082	0.801735466696407	2.109098336774	100	cis
outcome208	regressor2	0.0382702047011138	-0.682758612021832	-2.1005934479593	100	cis
outcome225	regressor2	0.024157062052826	-0.826899252917558	-2.29051070992782	100	cis
outcome228	regressor1	0.074119546844001	-0.717821214605752	-1.80536638555492	100	cis
outcome234	regressor1	0.0385051721096077	-0.774416981592756	-2.09798906238092	100	cis
outcome234	regressor2	0.0617757265232993	-0.717369945905024	-1.88976285410884	100	cis
outcome236	regressor1	0.0355381511503516	0.688156034855815	2.13193633159149	100	cis
outcome238	regressor2	0.0719879383077104	-0.593636714943202	-1.81905749747385	100	cis
outcome240	regressor2	0.0809272516278244	0.624164109029037	1.76372304837767	100	cis
outcome242	regressor1	0.0593578079286572	-0.654120576110962	-1.90792972639492	100	cis
outcome242	regressor2	0.0633526548922136	0.659882566811722	1.87823427759309	100	cis
outcome249	regressor1	0.00186564964167335	1.08965939305247	3.19862957903426	100	cis
outcome253	regressor2	0.0375872880608896	0.759595229206705	2.10824208765349	100	cis
outcome254	regressor1	0.0232851508089715	0.77101080257573	2.30522044701617	100	cis
outcome256	regressor2	0.0833915290168605	-0.56274181518592	-1.74936086733439	100	cis
outcome267	regressor1	0.0957437484335451	0.560835840522436	1.68221518773859	100	cis
outcome269	regressor2	0.071199095777629	0.701193762613245	1.82420968790928	100	cis
outcome271	regressor2	0.00466874374537275	0.971928182968585	2.89612753656533	100	cis
outcome273	regressor1	0.0671836684922851	0.700337371046884	1.85119794858801	100	cis
outcome276	regressor2	0.0604297003954238	-0.707893355698079	-1.89980070330654	100	cis
outcome282	regressor2	0.0459647254130645	-0.713265638405032	-2.02167286483455	100	cis
outcome28	regressor2	0.0360883560831539	-0.76993758420256	-2.12546050021042	100	cis
outcome301	regressor2	0.0932523645312674	0.668059055725962	1.6951634363025	100	cis
outcome302	regressor1	0.0932821614979764	0.598925004867524	1.69500691990512	100	cis
outcome305	regressor1	0.0646020911688394	0.663471592475476	1.86926970456509	100	cis
outcome309	regressor1	0.0820890738277998	-0.572606275905679	-1.75690754806559	100	cis
outcome30	regressor1	0.0490856586485422	0.695042911283764	1.99286642698083	100	cis
outcome316	regressor1	0.0518343849010651	-0.649371021101207	-1.96876098776617	100	cis
outcome316	regressor2	0.098765716903497	0.566345047432779	1.66687196626851	100	cis
outcome317	regressor2	0.0731471177284884	-0.658732191430364	-1.81157106092615	100	cis
outcome319	regressor2	0.0130601158732101	0.904615757893962	2.52874750871309	100	cis
outcome31	regressor1	0.0872555019013555	-0.633235007135622	-1.72752840137142	100	cis
outcome31	regressor2	0.0711562102680008	0.683775436742747	1.82449114641938	100	cis
outcome321	regressor1	0.0905895453461516	-0.602743759419177	-1.70931817675615	100	cis
outcome324	regressor2	0.0144995496368408	-0.920418225831289	-2.48935829280817	100	cis
outcome332	regressor1	0.0843358853233439	0.644680139086993	1.74394949137603	100	cis
outcome332	regressor2	0.0370791905200238	-0.794729593104039	-2.11401105192475	100	cis
outcome334	regressor1	0.071130471436237	0.656191880268571	1.82466013873563	100	cis
outcome338	regressor1	0.0691716453415685	-0.647014883591254	-1.83767312835552	100	cis
outcome348	regressor1	0.0367710797005825	-0.662202804161442	-2.11754273278119	100	cis
outcome354	regressor2	0.0122222288237196	-1.02938596575704	-2.55351020243285	100	cis
outcome35	regressor2	0.0551418146168607	0.764264031290512	1.94115531600462	100	cis
outcome362	regressor2	0.0405226710373019	-0.798206324377243	-2.07617408394147	100	cis
outcome369	regressor2	0.0950572929292468	0.656003012108766	1.68575521302923	100	cis
outcome370	regressor2	0.0108479680001617	-0.819372304976225	-2.59764510093428	100	cis
outcome382	regressor2	0.0915635272673252	0.616165972181425	1.70410188122424	100	cis
outcome385	regressor1	0.0278974947655065	-0.814535095366848	-2.23227266307482	100	cis
outcome387	regressor1	0.0920392501465234	0.610056482658421	1.70157052836902	100	cis
outcome393	regressor1	0.086664419791595	0.570144444150558	1.73081609501666	100	cis
outcome393	regressor2	0.00483282608975181	0.948184318745598	2.88431923838496	100	cis
outcome396	regressor1	0.0715307997628861	0.577206705738884	1.82203746129547	100	cis
outcome398	regressor2	0.00980074425854784	-0.92266273382437	-2.63480645358729	100	cis
outcome401	regressor1	0.0448774093457054	0.704056965468268	2.03210233428448	100	cis
outcome403	regressor1	0.0949079959271159	0.598037167668214	1.6865278836893	100	cis
outcome404	regressor2	0.0246402692466309	0.719102697271058	2.28255888642759	100	cis
outcome406	regressor2	0.0581107285169428	0.684072796814476	1.91754451445988	100	cis
outcome413	regressor2	0.00539303640656252	-1.0025357824627	-2.84660355163417	100	cis
outcome416	regressor2	0.0028787851736976	1.03267729144663	3.05811964829748	100	cis
outcome421	regressor2	0.0283811353508145	0.82968365835625	2.22525027296025	100	cis
outcome429	regressor1	0.090976517242947	0.679219389647362	1.70724023762796	100	cis
outcome434	regressor2	0.00950410699888686	0.967060627992084	2.6459858466748	100	cis
outcome436	regressor2	0.0862446738210344	-0.676824400319513	-1.73316197104814	100	cis
outcome439	regressor1	0.00244263565506609	-1.1118104628754	-3.11184362872866	100	cis
outcome443	regressor1	0.0976705686103898	0.625706923805376	1.67238768497562	100	cis
outcome446	regressor2	0.0709777755703579	0.690286194884683	1.82566373545026	100	cis
outcome451	regressor2	0.0756422517360437	0.650764929166659	1.79578500131138	100	cis
outcome457	regressor1	0.0466919359911347	0.686256299902367	2.01481467603275	100	cis
outcome458	regressor2	0.0984122061229453	-0.569054627899903	-1.6686469791053	100	cis
outcome462	regressor1	0.0249259744300418	0.86988379389916	2.27792153851597	100	cis
outcome466	regressor2	0.042134271379611	0.729152455397054	2.05941012201082	100	cis
outcome469	regressor1	0.0543320398103394	-0.745191202628327	-1.94778162904745	100	cis
outcome488	regressor1	0.0795536813200469	0.698407406649844	1.77188557564612	100	cis
outcome500	regressor1	0.066606790898461	-0.567208325129313	-1.85518518569835	100	cis
outcome506	regressor2	0.017505840259248	-0.859211237288858	-2.41727075271906	100	cis
outcome50	regressor2	0.0416855693176675	0.762043861941628	2.06402137440809	100	cis
outcome514	regressor2	0.040215685768861	0.70071871652173	2.07943223200634	100	cis
outcome521	regressor2	0.0616315899438287	0.695527164307664	1.89082889407806	100	cis
outcome525	regressor1	0.096191130905309	-0.541221321104425	-1.67991914462347	100	cis
outcome525	regressor2	0.0960687519810856	-0.554512268704448	-1.68054635026562	100	cis
outcome527	regressor1	0.0333480233298151	-0.765503571384406	-2.15860741001117	100	cis
outcome537	regressor2	0.0254964895092314	-0.849281770428151	-2.26879953483228	100	cis
outcome541	regressor2	0.0097978084573501	0.908247113494673	2.63491558609405	100	cis
outcome547	regressor2	0.0348568457495496	0.771917861508338	2.14007667600483	100	cis
outcome549	regressor1	0.022404540218787	-0.827670604920396	-2.32057835950317	100	cis
outcome549	regressor2	0.0393063314183071	-0.767002999768005	-2.08921088820268	100	cis
outcome554	regressor1	0.0788176549397714	0.609357509631882	1.77630743642216	100	cis
outcome559	regressor1	0.0456469616118042	0.793358395889526	2.02469880595906	100	cis
outcome563	regressor1	0.00214988277977765	1.11274805987269	3.15316067346017	100	cis
outcome568	regressor1	0.0988344830043935	-0.525023836838321	-1.66652728346477	100	cis
outcome569	regressor2	0.0359756796094691	0.843730694217078	2.1267796900358	100	cis
outcome56	regressor1	0.0402959705018643	0.81055253783075	2.07857807789851	100	cis
outcome56	regressor2	0.0220717424475499	0.92421354911673	2.32652139065011	100	cis
outcome570	regressor1	0.0406040681715929	-0.723229656236773	-2.07531375129772	100	cis
outcome576	regressor2	0.0941860970535985	-0.600342908587662	-1.69027800826017	100	cis
outcome578	regressor1	0.0191720966404658	-0.822660575336374	-2.38195230939882	100	cis
outcome582	regressor2	0.0672717961364965	-0.771389510118953	-1.85059135413336	100	cis
outcome590	regressor2	0.0529102600763073	-0.706811475045938	-1.95962113807125	100	cis
outcome592	regressor2	0.0898514141851914	0.620360695234162	1.71330194247566	100	cis
outcome595	regressor2	0.0228838343497157	-0.766215524555067	-2.31215472795885	100	cis
outcome606	regressor2	0.0782089953987702	0.678049093335922	1.77998995373371	100	cis
outcome609	regressor2	0.0966122434908707	0.665792312823235	1.67776583820608	100	cis
outcome618	regressor2	0.0123320842933579	0.878438837280246	2.55017816046665	100	cis
outcome624	regressor1	0.0291833582857381	0.694180700612051	2.21382966101501	100	cis
outcome625	regressor1	0.0607887586878741	0.652610591000957	1.89710481399916	100	cis
outcome626	regressor1	0.0261891549050969	0.734066343092088	2.25796246122452	100	cis
outcome626	regressor2	0.0724162197893984	-0.610156907072041	-1.81627989219432	100	cis
outcome628	regressor2	0.0268115949458054	0.895833022993212	2.24843661946292	100	cis
outcome631	regressor1	0.0456441659209966	0.758235124651876	2.02472550810776	100	cis
outcome641	regressor1	0.0431961163319957	-0.711759893676907	-2.0486634709692	100	cis
outcome642	regressor2	0.0557254142353228	-0.66394549605629	-1.93643066752958	100	cis
outcome644	regressor2	0.00161159165955761	1.06690702852739	3.24511603903958	100	cis
outcome652	regressor2	0.0791278507631177	0.577668069329008	1.77443971790861	100	cis
outcome659	regressor1	0.0942869529388091	0.595305648606902	1.68975267856399	100	cis
outcome65	regressor1	0.0377392984707929	0.736441742604972	2.10652925244842	100	cis
outcome666	regressor1	0.00948684961973644	-0.919674569217401	-2.64664590758893	100	cis
outcome672	regressor2	0.0290068021965837	-0.976260335296136	-2.21631938077906	100	cis
outcome674	regressor2	0.0159928863412391	0.796815354277694	2.45203541644515	100	cis
outcome676	regressor2	0.025313092466955	0.820646875417434	2.27171214443396	100	cis
outcome677	regressor1	0.059514091929529	0.636224029873335	1.9067368382637	100	cis
outcome685	regressor1	0.0590770701312405	0.675013536568182	1.91007921188377	100	cis
outcome694	regressor1	0.0581072703711124	0.620067438687918	1.91757141720645	100	cis
outcome697	regressor1	0.0278503267597961	0.754338537820734	2.23296322624559	100	cis
outcome700	regressor1	0.0581485068813774	0.72075370378167	1.91725070452401	100	cis
outcome706	regressor2	0.019498563050722	0.880314028645349	2.37535381993045	100	cis
outcome713	regressor1	0.0182182822626808	0.847503282156108	2.40181892446443	100	cis
outcome715	regressor2	0.0247079237742024	0.812741770932319	2.28145651268433	100	cis
outcome717	regressor2	0.0709573001873686	0.642093011089139	1.82579844697525	100	cis
outcome719	regressor2	0.0430898852258915	0.762040622442738	2.04972833361379	100	cis
outcome721	regressor1	0.0277857111224749	0.817298571713068	2.23391089557208	100	cis
outcome731	regressor1	0.0624689439754435	0.626106722551704	1.88466480441369	100	cis
outcome731	regressor2	0.0255468647781903	-0.7643214034427	-2.26800272585794	100	cis
outcome733	regressor2	0.0686692943970674	0.698583813422327	1.8410598257561	100	cis
outcome738	regressor1	0.0327551638144975	0.692311149825882	2.1660884865942	100	cis
outcome73	regressor2	0.0842641681853648	0.60403261226873	1.74435869430871	100	cis
outcome742	regressor2	0.083960113988022	0.600223359960549	1.74609674951238	100	cis
outcome744	regressor2	0.0768548128138085	0.693193936936765	1.78826883079575	100	cis
outcome749	regressor2	0.0343212485678663	-0.80955673927207	-2.14657351161662	100	cis
outcome755	regressor2	0.0972631964335332	0.613909767202534	1.67445222495851	100	cis
outcome757	regressor1	0.0273785989365423	-0.766665348031251	-2.23992648507453	100	cis
outcome764	regressor1	0.0764616122133907	-0.593328103134446	-1.79069529318273	100	cis
outcome769	regressor1	0.0330930528288611	0.764670780012477	2.16181050549123	100	cis
outcome772	regressor1	0.0626329509060885	-0.675515300371809	-1.88346562286492	100	cis
outcome780	regressor2	0.0452667915043014	-0.643744646156182	-2.02834279092629	100	cis
outcome782	regressor1	0.0136818107388532	-0.894838258957848	-2.51128024275341	100	cis
outcome793	regressor1	0.0956517124175538	-0.631868572462687	-1.68268861352272	100	cis
outcome794	regressor2	0.0354423233482276	-0.750243039166626	-2.13307307975257	100	cis
outcome800	regressor1	0.0415190719059487	0.800661574636795	2.06574330475783	100	cis
outcome802	regressor1	0.0790112548861104	0.61525608863718	1.77514104214803	100	cis
outcome805	regressor1	0.097120125620526	-0.631069184491094	-1.67517896566279	100	cis
outcome805	regressor2	0.0645060428535095	-0.717551486435289	-1.86995365887613	100	cis
outcome834	regressor2	0.0114797011859181	-0.871281843775121	-2.57676610146074	100	cis
outcome835	regressor1	0.0972222757481042	0.622406943451688	1.67465999649097	100	cis
outcome841	regressor2	0.0596964901670181	0.687665350113539	1.90534795779766	100	cis
outcome844	regressor2	0.0563548682167048	-0.761303975624677	-1.93138148740709	100	cis
outcome855	regressor1	0.0261412436301296	-0.736612040595617	-2.25870390400317	100	cis
outcome85	regressor2	0.0482818251114589	-0.750917461322071	-2.00013296109227	100	cis
outcome863	regressor2	0.0369770535410775	0.769004685099214	2.11517896264864	100	cis
outcome866	regressor2	0.0458422373901213	0.704871128099364	2.02283714474823	100	cis
outcome872	regressor1	0.011208605136082	-0.928923817168751	-2.5855957088647	100	cis
outcome874	regressor2	0.0449771976552938	0.690583655800641	2.03113620936407	100	cis
outcome878	regressor2	0.055229739263212	-0.719541924761969	-1.94044080483745	100	cis
outcome880	regressor2	0.0329860527109893	-0.679628458736463	-2.16316109926101	100	cis
outcome884	regressor2	0.044478030008858	0.673266757013661	2.03598753629773	100	cis
outcome885	regressor1	0.0208387017336197	0.759922001070323	2.34925424677012	100	cis
outcome886	regressor1	0.0800704325157051	-0.693041487129704	-1.76880120425506	100	cis
outcome88	regressor1	0.00984120192483462	-0.998119493220551	-2.63330552621073	100	cis
outcome891	regressor1	0.0151130281818426	-0.898979386418674	-2.47362768370506	100	cis
outcome898	regressor1	0.0208945173465982	-0.869480971730896	-2.34819981495113	100	cis
outcome899	regressor2	0.0294403613853267	-0.863761744824699	-2.21022893361297	100	cis
outcome90	regressor1	0.00090750811803128	1.07352688727986	3.42335277525591	100	cis
outcome911	regressor1	0.0116162486010068	0.918056558392799	2.5723896978881	100	cis
outcome912	regressor1	0.0662352064885692	-0.742672118846116	-1.85776880501597	100	cis
outcome913	regressor2	0.0313734775132565	0.778358212597586	2.18399139450485	100	cis
outcome917	regressor1	0.0413025216754945	0.789762644283235	2.06799179121301	100	cis
outcome920	regressor1	0.0177738178056346	0.814182474884086	2.41139416612064	100	cis
outcome924	regressor1	0.0645869429789218	0.568737473775325	1.86937751729	100	cis
outcome930	regressor2	0.0697921172758778	0.672287226939565	1.83351829878521	100	cis
outcome938	regressor1	0.0147328344411367	-0.865309933468076	-2.48330753112593	100	cis
outcome939	regressor2	0.0830277301234489	-0.665449714042629	-1.75145897001438	100	cis
outcome944	regressor1	0.0956421351182212	-0.648147158601286	-1.6827378996443	100	cis
outcome948	regressor1	0.0838795925830087	0.583877523835293	1.74655789784555	100	cis
outcome94	regressor1	0.0171168484526506	-0.840422497436936	-2.42594524284537	100	cis
outcome951	regressor2	0.0661269837869562	-0.609461348425244	-1.85852355521466	100	cis
outcome955	regressor1	0.0759323398696888	-0.583084399620283	-1.79397783051151	100	cis
outcome95	regressor2	0.0724197680466057	0.619879844688312	1.81625693720836	100	cis
outcome971	regressor2	0.0403286941424766	-0.728242887420865	-2.07823034947959	100	cis
outcome973	regressor1	0.0991232194565037	0.516404006189337	1.66508214087076	100	cis
outcome980	regressor2	0.0749767665397897	0.653231157732727	1.79995260027153	100	cis
outcome984	regressor2	0.0291154436600576	-0.741335768750556	-2.21478580251854	100	cis
outcome987	regressor2	3.34421585937524e-05	-1.54762457595851	-4.35172639450052	100	cis
outcome990	regressor1	0.0333748431954378	-0.790143808425353	-2.15827172058941	100	cis
outcome990	regressor2	0.0357164044519315	-0.798897188802006	-2.12982886441803	100	cis
outcome999	regressor1	0.0533839347665889	0.669882964807523	1.95564707942863	100	cis
outcome9	regressor1	0.0848730090154889	-0.613977978043623	-1.74089383244991	100	cis
